app/jobs/api/issues_label_issue_job.rb

```diff
@@ -1,11 +1,18 @@
 module Api
   class IssuesLabelIssueJob < IssueEventJob
-    include IsPublishable
     timestamp ->(params) { params[:issue]['updated_at'] }
     action "issue_labeled"
+    cache_key ->(message) {
+      label_name = message[:payload][:label] ? message[:payload][:label]['name'] : ""
+
+      "#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}.#{ label_name }"
```

Comment:
Are there any issue messages that wouldn't be cached with at least `#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}`? Could the override just specify a suffix for the key if necessary, to avoid duplicating the action/login/id/timestamp code?